### ISSUE TITLE
Results of this model.

### ISSUE THREAD

**Issue by Mr-Nobody-dey** [mapooon/BlendFace#11]
The swap result of this model is not good.
The similarity of the source face with the swapped face is very poor.

**Note that-> I have used proper alignment methods as mentioned in this repository.**

I will upload the similarity matrix and result later.

**Comment by linzixuan45** [comment]
I also encountered the same problem.  I tried to use BlendFace weights to train the face swap model, but the results showed that the identity similarity between Source and Swap was low. The swap matrix results during training were also not very good (as shown below. The left side is the Target, and the upper side is the Source).  In addition, I set the training step to 50,000 steps, batch size to 16, and learning rate to 0.0001.

**Comment by linzixuan45** [comment]
![Uploading step_49200.jpg…]()

**Comment by linzixuan45** [comment]
![step_49200](https://github.com/mapooon/BlendFace/assets/63531979/03ba7675-ae13-40bc-9dd3-f776c4459662)

**Comment by YoucanBaby** [comment]
same question
